platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/AddressBookInitializerTest.java

```diff
@@ -482,18 +485,20 @@ private void assertAddressBookFileContent(
     }
 
     /**
-     * Creates an AddressBookConfig object with the given forceUseOfConfigAddressBook value.
+     * Creates an PlatformContext object with a configuration able to return AddressBookConfiguration with the given
+     * forceUseOfConfigAddressBook value.
      *
      * @param forceUseOfConfigAddressBook the value to use for the forceUseOfConfigAddressBook property.
-     * @return the AddressBookConfig object.
+     * @return the PlatformContext object.
      */
-    private AddressBookConfig getAddressBookConfig(boolean forceUseOfConfigAddressBook) {
+    private PlatformContext getPlatformContext(boolean forceUseOfConfigAddressBook) {
```

Comment:
We already have `TestPlatformContextBuilder` that should be used here.

Reply:
replaced the implementation of `getPlatformContext()` to use the `TestPlatformContextBuilder`